packages/react-components/src/components/chart/useTrendCursors.ts

```diff
@@ -47,10 +58,6 @@ const useTrendCursors = (
       }
     };
   }, [ref, graphic, size, isInCursorAddMode, setGraphic, viewport, theme, series, yMin, yMax]);
-
-  useEffect(() => {
```

Comment:
nice catch! i was looking everywhere to find where the console log was coming from 😭